src/core/importType.js

```diff
@@ -49,6 +49,10 @@ function isRelativeToSibling(name) {
   return name.indexOf('./') === 0
 }
 
+export function isPrivateModule(name) {
+  return name.indexOf('@') === 0
```

Comment:
note that this might not actually be true - I could make a folder called "@foo" and put a package.json in it, and `require('@foo')` would work, but it wouldn't be a private module.